contrib/platform/src/com/sun/jna/platform/win32/Advapi32Util.java

```diff
@@ -1758,13 +1758,14 @@ public EnumKey(HKEY hKey, int dwIndex) {
 	 * @return A environment block
 	 */
 	public static String getEnvironmentBlock(Map<String, String> environment) {
-		StringBuilder out = new StringBuilder();
+		StringBuilder out = new StringBuilder(environment.size() * 32 /* some guess about average name=value length*/);
 		for (Entry<String, String> entry : environment.entrySet()) {
-			if (entry.getValue() != null) {
-				out.append(entry.getKey() + "=" + entry.getValue() + "\0");
+		    String    key=entry.getKey(), value=entry.getValue();
```

Comment:
I overlooked this before, it's not that important, but it looks sloppy to future developers - I'll take it, but I would appreciate if you could look into auto-formatting this and others (I don't have the exact Eclipse configuration, sorry), and maybe run something automatic for a future commit. We would like to make JNA look like it was written by 1 person.